.github/workflows/c3-e2e.yml

```diff
@@ -5,6 +5,11 @@ on:
   pull_request:
     paths:
       - packages/create-cloudflare/**
+  push:
+    branches:
+      - main
+    paths:
+      - packages/create-cloudflare/**
```

Comment:
Would it be possible for a change in wrangler to break these?

Reply:
Good question!
I think it is unlikely that a wrangler change that doesn't break some other fixture tests would break these. So I think we are good on coverage. And in the worst case they would fail the next time these get run as part of a change to C3. The downside of these running all the time outweighs that I think.